src/test/java/ru/ayaz/UserIgnoredSetTest.java

```diff
@@ -0,0 +1,44 @@
+package ru.ayaz;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+public class UserIgnoredSetTest {
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    private User user;
+
+    @Before
+    public final void before() {
+        user = new User();
+        user.setNickname("Ayaz");
+    }
+
+    @Test
+    public void notNullPointerExceptionWhenInitialized() {
+        user.hasInIgnoredSet("someone");
```

Comment:
Yes, I thought it would be like "User has...", but function name should reference to argument and it should look like "Someone is..."